2022-05-30 17:15:51 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "create table CORSO (\000d\000a    codice INTEGER primary key,\000d\000a    denominazione VARCHAR(50) not null,\000d\000a    CFU INTEGER not null,\000d\000a[*])"; expected "identifier"; SQL statement:
create table CORSO (
    codice INTEGER primary key,
    denominazione VARCHAR(50) not null,
    CFU INTEGER not null,
) [42001-212]
2022-05-30 17:15:51 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "CORSO" not found; SQL statement:
create table TITOLARITA (
    codiceCorso INTEGER references CORSO(codice),
    matricolaDocente CHAR(9) references DOCENTE(matricola),
    annoAccademico INTEGER not null,
    constraint pk_acd primary key (codiceCorso, matricolaDocente)
) [42102-212]
2022-05-30 17:15:51 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "create table APPELLO (\000d\000a    id INTEGER primary key,\000d\000a    data DATE not null,\000d\000a    scadenza DATE not null,\000d\000a    note VARCHAR(500),\000d\000a    sede VARCHAR(50) check (sede in ('AULA', 'LABORATORIO', 'ALTRO')),\000d\000a    codiceCorso INTEGER references CORSO(codice),\000d\000a    matricolaDocente INTEGER references DOCENTE(matricola),\000d\000a    constraint check_scadenza check (scadenza >= data),\000d\000a[*])"; expected "identifier"; SQL statement:
create table APPELLO (
    id INTEGER primary key,
    data DATE not null,
    scadenza DATE not null,
    note VARCHAR(500),
    sede VARCHAR(50) check (sede in ('AULA', 'LABORATORIO', 'ALTRO')),
    codiceCorso INTEGER references CORSO(codice),
    matricolaDocente INTEGER references DOCENTE(matricola),
    constraint check_scadenza check (scadenza >= data),
) [42001-212]
2022-05-30 17:15:51 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "APPELLO" not found; SQL statement:
create table VERBALE (
    id INTEGER primary key,
    appello INTEGER references APPELLO(id)
) [42102-212]
2022-05-30 17:15:51 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "APPELLO" not found; SQL statement:
create table ESAME (
    voto INTEGER not null,
    argomenti_trattati VARCHAR(500),
    appello INTEGER references APPELLO(id),
    matricolaStudente INTEGER references STUDENTE(matricola),
    constraint check_voto check (voto >= 0 and voto <= 30),
    constraint pk primary key (appello, matricolaStudente)
) [42102-212]
2022-05-30 17:15:51 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "APPELLO" not found; SQL statement:
create table PRENOTAZIONE (
    matricolaStudente CHAR(9) references STUDENTE(matricola),
    appello INTEGER references APPELLO(id),
    constraint pk primary key (matricolaStudente, appello)
) [42102-212]
2022-05-30 17:18:55 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "drop [*]docente"; expected "TABLE, INDEX, USER, SEQUENCE, CONSTANT, TRIGGER, VIEW, ROLE, ALIAS, SCHEMA, ALL, DOMAIN, TYPE, DATATYPE, AGGREGATE, SYNONYM"; SQL statement:
drop docente [42001-212]
2022-05-30 17:19:23 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "create table CORSO (\000d\000a    codice INTEGER primary key,\000d\000a    denominazione VARCHAR(50) not null,\000d\000a    CFU INTEGER not null,\000d\000a[*])"; expected "identifier"; SQL statement:
create table CORSO (
    codice INTEGER primary key,
    denominazione VARCHAR(50) not null,
    CFU INTEGER not null,
) [42001-212]
2022-05-30 17:20:44 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Constraint "PK" already exists; SQL statement:
create table PRENOTAZIONE (
    matricolaStudente CHAR(9) references STUDENTE(matricola),
    appello INTEGER references APPELLO(id),
    constraint pk primary key (matricolaStudente, appello)
) [90045-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:109)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:72)
	at org.h2.command.ddl.AlterTable.update(AlterTable.java:46)
	at org.h2.command.ddl.CommandWithColumns.createConstraints(CommandWithColumns.java:83)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:131)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:832)
2022-05-30 17:21:58 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Cannot drop "APPELLO" because "CONSTRAINT_3FB, CONSTRAINT_3F3, CONSTRAINT_FB74" depends on it; SQL statement:
drop table appello [90107-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:97)
	at org.h2.command.ddl.DropTable.update(DropTable.java:121)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:832)
2022-05-30 17:21:58 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Cannot drop "CORSO" because "CONSTRAINT_5, CONSTRAINT_FB8" depends on it; SQL statement:
drop table corso [90107-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:97)
	at org.h2.command.ddl.DropTable.update(DropTable.java:121)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:832)
2022-05-30 17:21:58 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Cannot drop "DOCENTE" because "CONSTRAINT_5D, CONSTRAINT_FB80" depends on it; SQL statement:
drop table docente [90107-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:97)
	at org.h2.command.ddl.DropTable.update(DropTable.java:121)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:832)
2022-05-30 17:22:09 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Cannot drop "CORSO" because "CONSTRAINT_FB8" depends on it; SQL statement:
drop table corso [90107-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:97)
	at org.h2.command.ddl.DropTable.update(DropTable.java:121)
	at org.h2.command.CommandContainer.update(CommandContainer.java:174)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:252)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:832)
2022-05-30 17:22:32 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "create table APPELLO (\000d\000a    id INTEGER primary key,\000d\000a    data DATE not null,\000d\000a    scadenza DATE not null,\000d\000a    note VARCHAR(500),\000d\000a    sede VARCHAR(50) check (sede in ('AULA', 'LABORATORIO', 'ALTRO')),\000d\000a    codiceCorso INTEGER references CORSO(codice),\000d\000a    matricolaDocente INTEGER references DOCENTE(matricola),\000d\000a    constraint check_scadenza check (scadenza >= data),\000d\000a[*])"; expected "identifier"; SQL statement:
create table APPELLO (
    id INTEGER primary key,
    data DATE not null,
    scadenza DATE not null,
    note VARCHAR(500),
    sede VARCHAR(50) check (sede in ('AULA', 'LABORATORIO', 'ALTRO')),
    codiceCorso INTEGER references CORSO(codice),
    matricolaDocente INTEGER references DOCENTE(matricola),
    constraint check_scadenza check (scadenza >= data),
) [42001-212]
2022-05-30 17:22:32 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "APPELLO" not found; SQL statement:
create table VERBALE (
    id INTEGER primary key,
    appello INTEGER references APPELLO(id)
) [42102-212]
2022-05-30 17:22:32 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "APPELLO" not found; SQL statement:
create table ESAME (
    voto INTEGER not null,
    argomenti_trattati VARCHAR(500),
    appello INTEGER references APPELLO(id),
    matricolaStudente INTEGER references STUDENTE(matricola),
    constraint check_voto check (voto >= 0 and voto <= 30),
    constraint pk_esame primary key (appello, matricolaStudente)
) [42102-212]
2022-05-30 17:22:32 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "APPELLO" not found; SQL statement:
create table PRENOTAZIONE (
    matricolaStudente CHAR(9) references STUDENTE(matricola),
    appello INTEGER references APPELLO(id),
    constraint pk_prenotazione primary key (matricolaStudente, appello)
) [42102-212]
2022-05-30 17:38:57 jdbc[5]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-212]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:190)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:132)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:288)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:808)
	at org.h2.server.web.WebApp.login(WebApp.java:1033)
	at org.h2.server.web.WebApp.process(WebApp.java:226)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:176)
	at org.h2.server.web.WebThread.process(WebThread.java:153)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:832)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: The connection was not closed by the application and is garbage collected [90018-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 13 more
2022-05-31 14:01:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "N46004793" not found; SQL statement:
insert into studente values ("N46004793", "Cirillo", "Francesco", "SirFrigo", "password", "11111") [42122-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.doPrepare(Insert.java:300)
	at org.h2.command.dml.DataChangeStatement.prepare(DataChangeStatement.java:37)
	at org.h2.command.Parser.prepareCommand(Parser.java:574)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:832)
2022-05-31 14:01:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "N4600479" not found; SQL statement:
insert into studente values ("N4600479", "Cirillo", "Francesco", "SirFrigo", "password", "11111") [42122-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.doPrepare(Insert.java:300)
	at org.h2.command.dml.DataChangeStatement.prepare(DataChangeStatement.java:37)
	at org.h2.command.Parser.prepareCommand(Parser.java:574)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:832)
2022-05-31 14:01:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "N46004793" not found; SQL statement:
insert into studente values ("N46004793", "Cirillo", "Francesco", "SirFrigo", "password", "11111") [42122-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.doPrepare(Insert.java:300)
	at org.h2.command.dml.DataChangeStatement.prepare(DataChangeStatement.java:37)
	at org.h2.command.Parser.prepareCommand(Parser.java:574)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:832)
2022-05-31 14:02:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "N46004793" not found; SQL statement:
insert into studente (matricola, cognome, nome, username, password, pin) values ("N46004793", "Cirillo", "Francesco", "SirFrigo", "password", "11111") [42122-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.doPrepare(Insert.java:300)
	at org.h2.command.dml.DataChangeStatement.prepare(DataChangeStatement.java:37)
	at org.h2.command.Parser.prepareCommand(Parser.java:574)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:832)
2022-05-31 14:03:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "N46004793" not found; SQL statement:
insert into studente values ("N46004793", "Cirillo", "Francesco", "SirFrigo", "password", "11111") [42122-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.doPrepare(Insert.java:300)
	at org.h2.command.dml.DataChangeStatement.prepare(DataChangeStatement.java:37)
	at org.h2.command.Parser.prepareCommand(Parser.java:574)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:832)
2022-05-31 14:03:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "N46004793" not found; SQL statement:
insert into STUDENTE values ("N46004793", "Cirillo", "Francesco", "SirFrigo", "password", "11111") [42122-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.doPrepare(Insert.java:300)
	at org.h2.command.dml.DataChangeStatement.prepare(DataChangeStatement.java:37)
	at org.h2.command.Parser.prepareCommand(Parser.java:574)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:832)
2022-05-31 14:03:41 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "N46004793" not found; SQL statement:
insert into studente values ("N46004793", "Cirillo", "Francesco", "SirFrigo", "password", "11111") [42122-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.doPrepare(Insert.java:300)
	at org.h2.command.dml.DataChangeStatement.prepare(DataChangeStatement.java:37)
	at org.h2.command.Parser.prepareCommand(Parser.java:574)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:832)
2022-05-31 14:06:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "N46004793" not found; SQL statement:
insert into STUDENTE values ("N46004793", "Cirillo", "Francesco", "SirFrigo", "password", "11111") [42122-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.doPrepare(Insert.java:300)
	at org.h2.command.dml.DataChangeStatement.prepare(DataChangeStatement.java:37)
	at org.h2.command.Parser.prepareCommand(Parser.java:574)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:832)
2022-05-31 14:06:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "Cirillo" not found; SQL statement:
insert into STUDENTE values ('N46004793', "Cirillo", "Francesco", "SirFrigo", "password", "11111") [42122-212]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.command.dml.Insert.doPrepare(Insert.java:300)
	at org.h2.command.dml.DataChangeStatement.prepare(DataChangeStatement.java:37)
	at org.h2.command.Parser.prepareCommand(Parser.java:574)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:237)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:223)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1339)
	at org.h2.server.web.WebApp.query(WebApp.java:1137)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1103)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1090)
	at org.h2.server.web.WebThread.process(WebThread.java:189)
	at org.h2.server.web.WebThread.run(WebThread.java:102)
	at java.base/java.lang.Thread.run(Thread.java:832)
2022-05-31 14:22:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "drop [*]appello, corso, docente, prenotazione, studente, titolarita, valutazione, verbale"; expected "TABLE, INDEX, USER, SEQUENCE, CONSTANT, TRIGGER, VIEW, ROLE, ALIAS, SCHEMA, ALL, DOMAIN, TYPE, DATATYPE, AGGREGATE, SYNONYM"; SQL statement:
drop appello, corso, docente, prenotazione, studente, titolarita, valutazione, verbale [42001-212]
2022-05-31 14:23:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "select * [*]froms studente"; SQL statement:
select * froms studente [42000-212]
